samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/MeetingLiveCoding.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Dotnet 3.1 is ending support in December of this year, should we use and test in dotnet 6 instead?

Reply:
We will create a separate task to migrate all the samples to latest version.